src/js/hexbin/HexbinLayer.js

```diff
@@ -302,10 +302,30 @@ L.HexbinLayer = (L.Layer ? L.Layer : L.Class).extend({
 		this.options.onclick = clickFn;
 		//this._redraw();
 		return this;
+	},
+
+	/*
+	 * Returns an array of the points in the path, or nested arrays of points in case of multi-polyline.
+	 */
+	getLatLngs: function () {
+		return this._data.map(function(d) { return L.latLng(d[0], d[1]); });
+	},
+
+	/*
+	 * Get path geometry as GeoJSON
+	 */
+	toGeoJSON: function () {
+		var latLngs = this._data.map(function(d) { return L.latLng(d[0], d[1]); });
```

Comment:
You should be able to call this.getLatLngs() here to avoid the redundant call to generate the latLng array. Also, with respect to the line below, can it ever be a multi geometry since we're generating the array of latLngs immediately above?